high_card.rb

```diff
@@ -1,5 +1,27 @@
 # Use these two arrays to generate a deck of cards.
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
+cardsdeck = []
 
+
+ranks.each do |rank|
+  suits.each do |suits|
+    cardsdeck.push (ranks, suits)
+  end
+
+end
+
+
+cardsdeck = playing_card_game
```

Comment:
What is `playing_card_game`? You're calling a variable that doesn't exist. This is going to throw an error.